### PR TITLE
💄 🐷 Test consistency

### PR DESCRIPTION
**Pull request by ErisDS**
- change out should.equal for // jshint ignore:line
- ensure should is the first require in every test, and ALWAYS require
- make sinon the second require, and sandbox the last thing
- ALWAYS use sandbox, futureproofs tests against contributors who don't know it
- change require formatting

Most of these changes just make it easy to grok that the test structure is correct. I keep finding files that didn't have should! I'm sure it'll go wrong again, but for now 👍

### CODE REVIEW

File: core/server/views/default.hbs [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<!doctype html>
```
Comment (acburdine) [inline core/server/views/default.hbs]: :trollface:

Comment (acburdine) [inline core/server/views/default.hbs]: thinking about it -> this should probably be added to the .gitignore

**Review by kirrg001 — COMMENTED** [review]
👍 

What we also can do is, we add a script in `core/test`, which will require all the basic test modules we need (should, sinon) and assign as globals. Then we tell mocha to load this script before the tests are executed, here https://github.com/TryGhost/Ghost/blob/master/Gruntfile.js#L147

Each test will have access to the test modules by default.

**Comment by ErisDS** [comment]
Yeah I'd also like to setup a global called `serverRequire` to reduce the ../../../  👍

**Comment by ErisDS** [comment]
To be clear, I'm not doing those things in this PR! 😁

**Review by kirrg001 — APPROVED** [review]
yeah i know 👍 Just waited for the admin view update.

**Pull request merged**